### PR TITLE
Component/dashboard

### LINKED ISSUE #25: [UI] Create the Dashboard
1. Create reusable `Dashboard` component
2. It should be able to get the DATA from the Parent (outside)
3. `Dashboard Header`
4. `Dashboard  Item`

### PR DESCRIPTION
**Pull request by pallabez**
# Description

image
Fixes #25

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Manual Testing
- [x] Unit Test 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ui/src/Components/Dashboard/TableRow/TableRow.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      className='border-t'
+    >
+      {headers.map((heading, index) =>
+        <td className='px-4 py-2' key={index}>{featureFlag[heading]}</td>
+      )}
+      <td className='px-4 py-2'>
+        <Toggle 
+          active={featureFlag.enabled}
+        />
+      </td>
```
Comment (pallabez) [inline ui/src/Components/Dashboard/TableRow/TableRow.js]: Should this be passed from props? This column will include toggle/edit/delete

File: ui/package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "autoprefixer": "^10.4.7",
     "babel-eslint": "^10.1.0",
     "postcss": "^8.4.14",
+    "sinon": "^14.0.0",
```
Comment (Neha) [inline ui/package.json:43]: Why we have added this?

File: ui/src/App.js [hunk 3]
```diff
@@ -1,6 +1,16 @@
+import Dashboard from "./Components/Dashboard/Dashboard";
+import { FEATURE_FLAGS } from "./test/Fixtures/featureFlags";
```
Comment (Neha) [inline ui/src/App.js]: Mock data should not be the part of the test folder

File: ui/src/App.js [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+
 function App() {
   return (
-   <h1 className="text-3xl font-bold underline">Hey</h1>
+    <main 
+      className='flex flex-col items-center'
+    >
+      <Dashboard 
+        featureFlags={FEATURE_FLAGS} 
+        headers={['featureFlag', 'repository']}
```
Comment (Neha) [inline ui/src/App.js]: This shouldn't be hardcoded. Move to separate file as mock data

File: ui/src/Components/Dashboard/TableRow/TableRow.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import Toggle from "../../Global/Toggle/Toggle"
+
+export default function TableRow({ featureFlag, headers }) {
+  return (
+    <tr
+      className='border-t'
+    >
+      {headers.map((heading, index) =>
+        <td className='px-4 py-2' key={index}>{featureFlag[heading]}</td>
```
Comment (Neha) [inline ui/src/Components/Dashboard/TableRow/TableRow.js]: I am assuming this is WIP as of now. He need to add the tags for the text and that will depend on the type of content we will have in the table

File: ui/src/test/Fixtures/featureFlags.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+export const FEATURE_FLAGS = [
+  {
+    id: 1,
```
Comment (Neha) [inline ui/src/test/Fixtures/featureFlags.js]: We need more data here...such as created by , created when, modified etc...

**Review by Neha — CHANGES_REQUESTED** [review]

File: ui/src/Components/Dashboard/TableRow/TableRow.js (hunk 5 above)
Comment (pallabez) [reply ui/src/Components/Dashboard/TableRow/TableRow.js]: Sorry. I'm not getting what you mean.

Comment (pallabez) [reply ui/src/Components/Dashboard/TableRow/TableRow.js]: So, what we are planning is that we pass tags as props to enable that particular column?

**Pull request merged**